inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1_to_mvn6.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+#include <numeric>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1ToMVN6, "ConvertMVN1ToMVN6", 0);
+
+ngraph::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
+    auto mvn = pattern::wrap_type<ngraph::op::v0::MVN>();
```

Comment:
We specifically changed from specifying opsets to specifying specific op version since this is that kind of transformation, they transform specific versions of operations.

Reply:
We have some internal discussion and the decision was made to use opset version everywhere.

Reply:
@mvafin , sorry for my original incorrect comment in the review.

Reply:
@mvafin why you still use op::v0::MVN?